File: atividade3.c
#include <stdio.h>

int main() {
    float t_celsius, t_fahrenheit;

    printf("digite a temperatura em c°: ");
    scanf("%f", &t_celsius);

    t_fahrenheit = (t_celsius * 9 / 5) + 32;

    printf("%.2f graus celsius são%.2f graus farenheit.\n", t_celsius, t_fahrenheit);

    return 0;
}

File: atividade3.java
import java.util.Scanner;

public class atividade3 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Digite a temperatura em graus celsius ");
        double tempC = scanner.nextDouble();
        double tempF = (tempC * 9 / 5) + 32;
        System.out.println("A temperatura em Fahrenheit é " + tempF);

        scanner.close();
    }
}
